src/main/resources/templates/db_scripts/schema.json

```diff
@@ -738,16 +670,6 @@
         }
       ],
       "fullTextIndex": [
-        {
-          "fieldName": "callNumberAndSuffixNormalized",
-          "sqlExpression" : "normalize_holdings_call_number_and_suffix(jsonb)",
-          "sqlExpressionQuery": "normalize_call_number_string($) || '*'"
-        },
-        {
-          "fieldName": "fullCallNumberNormalized",
-          "sqlExpression" : "normalize_holdings_full_call_number(jsonb)",
-          "sqlExpressionQuery": "normalize_call_number_string($) || '*'"
```

Comment:
Can createCallNumberNormalizationFunctions.sql been deleted? And a dropCallNumberNormalizationFunctions.sql been added to schema.json that removes the functions?